js/apps/admin-ui/src/components/users/UserDataTable.tsx

```diff
@@ -391,6 +424,11 @@ export function UserDataTable() {
             displayKey: "firstName",
             cellFormatters: [emptyFormatter()],
           },
+          {
+            name: "Enable",
```

Comment:
The name here should use the same casing as other entries.
```suggestion
            name: "enable",
```